README.md

```diff
@@ -41,7 +41,7 @@ Let’s see how we can use aixplain to run a machine translation model.
 ```python
 from aixplain.factories.model_factory  import ModelFactory
 model = ModelFactory.get("61b27086c45ecd3c10d0608c") # Got the ID of an MT model from on our platform
-translation = model.run("This is a sample text")
+translation = model.run("This is a sample text") # Alternatively, you can input a URL or provide a file path to a text file
```

Comment:
Correct to: Alternatively, you can input a public URL or provide a file path on your local machine.